strum_macros/src/helpers/metadata.rs

```diff
@@ -242,15 +243,22 @@ impl Parse for VariantMeta {
     }
 }
 
-struct Prop(Ident, LitStr);
+struct Prop(Ident, PropertyValue);
 
 impl Parse for Prop {
     fn parse(input: ParseStream) -> syn::Result<Self> {
         use syn::ext::IdentExt;
 
         let k = Ident::parse_any(input)?;
         let _: Token![=] = input.parse()?;
-        let v = input.parse()?;
+        let v;
+        if input.peek(syn::LitBool) {
```

Comment:
Can we implement `Parse` for `PropertyValue` then just `let v: PropertyValue = input.parse()?` ?